VimeoNetworking/Sources/Models/VIMConnection.m

```diff
@@ -61,6 +61,7 @@
 NSString *const VIMConnectionNameNotifications = @"notifications";
 NSString *const VIMConnectionNameBlockUser = @"block";
 NSString *const VIMConnectionNameLiveStats = @"live_stats";
+NSString *const VIMConnectionNameTextTracks = @"texttracks";
```

Comment:
@Jumhyn just double checking - API doesn't send this as `text_tracks`?

Reply:
@nicolelehrer Nope. Silly, but just double-checked to make sure. It's `texttracks`.